src/accessibility-statement.md

```diff
@@ -97,11 +97,11 @@ DAC also tested the global search functionality that appears in the header of th
 
 ### Frontend documentation website
 
-The [GOV.UK Frontend documentation website](http://frontend.design-system.service.gov.uk/) was last audited for accessibility issues by an external group in April 2021. The audit was carried out by the [Digital Accessibility Centre (DAC)](https://digitalaccessibilitycentre.org/).
+The [GOV.UK Frontend documentation website](https://frontend.design-system.service.gov.uk/) was last audited for accessibility issues by an external group in April 2021. The audit was carried out by the [Digital Accessibility Centre (DAC)](https://digitalaccessibilitycentre.org/).
 
-DAC tested the [Technical Documentation Template](https://tdt-documentation.london.cloudapps.digital/), which is the template for the GOV.UK Frontend documentation website.
+DAC tested the [Technical Documentation Template](https://github.com/alphagov/tdt-documentation/), which is the template for the GOV.UK Frontend documentation website.
 
-To learn more, read the [Technical Documentation Template accessibility statement](https://tdt-documentation.london.cloudapps.digital/accessibility/).
+To learn more, read the [Technical Documentation Template accessibility statement](https://github.com/alphagov/tdt-documentation/blob/main/source/accessibility/index.html.md.erb).
```

Comment:
Added a small note that link will go into GitHub repo.

I understand that there's no website for this document any more, but it's pretty unexpected to be thrown into viewing the file within Github repo.

```suggestion
To learn more, read the [Technical Documentation Template accessibility statement in its GitHub repository](https://github.com/alphagov/tdt-documentation/blob/main/source/accessibility/index.html.md.erb).
```